wrapper/src/main/java/software/amazon/jdbc/hostlistprovider/RdsMultiAzDbClusterListProvider.java

```diff
@@ -187,4 +187,9 @@ private HostSpec createHost(final ResultSet resultSet, final String writerNodeId
     hostSpec.setHostId(hostId);
     return hostSpec;
   }
+
+  @Override
+  public String getClusterId() throws UnsupportedOperationException {
+    throw new UnsupportedOperationException("RdsMultiAzDbClusterListProvider does not support `getClusterId` method.");
```

Comment:
I'd expect its supported by this provider.

Reply:
Done!